app.py

```diff
@@ -357,10 +357,10 @@ def parse_error_to_response(ex, translations):
 arguments_that_require_translation = ['allowed_types', 'invalid_type', 'required_type', 'character_found', 'concept']
 
 def hedy_error_to_response(ex, translations):
-    error_message = translate_error(ex.error_code, translations, ex.arguments)
+    #error_message = translate_error(ex.error_code, translations, ex.arguments)
     location = ex.location if hasattr(ex, "location") else None
-    return {"Error": error_message, "Location": location}
-
+    invalid_command = ex.invalid_command if hasattr(ex, "invalid_command") else None
+    return {"Error": "error_message", "Location": location, "Invalid_command": invalid_command}
```

Comment:
I think you are trying to add some fields into the JSON here if they exist on the error, but in the process you turned off error translation for all errors. That's probably not what you intended to do.

I would recommend leaving the existing `Error` field to be what it is: a human-readable string that will be used for various purposes. 

-------------

Something else: I'm seeing you are adding the `Invalid_command` field to the response... but you are only using it to later on get the **length** of the invalid command, so that you can use it to mark a subsection of a line.

But what if we want the same feature later on to indicate type errors? Or where kids still need to fill in a placeholder? Are we going to reuse `Invalid_command`? Or are we going to add fields `Invalid_type`, `Invalid_placeholder`, etc? When all we need is just the length?

To keep it nice and simple, I would call the field something like `LocationLength`, and if given it will indicate the number of characters starting at `Location` that need to be highlighted because they contain a mistake.

If you do that, the code could look something like this:

```python
def hedy_error_to_response(ex, translations):
    return {
        "Error": translate_error(ex.error, translations, ex.arguments),
        "Location": getattr(ex, "location", None),
        "LocationLength": getattr(ex, "location_length", None),
    }
```